Update README.md (#2)
wso2/choreo-sample-todo-list-app: README.md

> # Next.js Todo List Application

Reviewer: ```suggestion
# Next.js To Do List Application
```

> This is a simple fullstack Next.js Todo list application that can be used to demonstrate WebApp deployments of Choreo. This application utilizes Typescript, React and Tailwind CSS

Reviewer: ```suggestion
This is a simple full stack `Next.js` to do list application that can be used to demonstrate web application deployment in Choreo. This application utilizes `Typescript`, `React` and `Tailwind CSS`.
```

> ## Deploy In Choreo

Reviewer: ```suggestion
## Deploy in Choreo
```

- Fork this repository.

> - Please refer to the Choreo documentation under the [Develop a Web Application](https://wso2.com/choreo/docs/develop-components/develop-a-web-application/) section to learn how to deploy the application.

Reviewer: Suggestion:
For instructions on how to deploy the web application, see [Deploy Your First Static Web Application](https://wso2.com/choreo/docs/quick-start-guides/deploy-your-first-static-web-application/) in the Choreo documentation.
